hppc/src/main/java/com/carrotsearch/hppc/WormUtil.java

```diff
@@ -0,0 +1,84 @@
+package com.carrotsearch.hppc;
+
+/**
+ * Standard Java hash for primitives.
+ * <p/>It is different from MapUtil because here we want <b>standard Java</b> implementations.
```

Comment:
The description was changed

Reply:
Speaking about BitMixer, I am not quite sure, it's better to ask Bruno about it

Reply:
This javadoc remains from old time. Yes there is no need to keep that.
About the BitMixer, we should use it for the hash salt different between each map instance. And I'd like to study the hash function distribution itself to compare it to the simple one we use.

Reply:
These hash functions have been through several iterations of refinement already. I didn't come up with these - Sebastiano Vigna did (fastutil). He's a great fellow to talk to if you're interested (and you may want to check out his research paper curriculum).

This doesn't mean nothing can be improved, I imagine. I'm just saying there's already some historical work put into this.

Reply:
Actually the one we use here is taken from FastUtil :)
I've just made a tool to try to measure the distribution and resilience to clustering effect. So I'll simply cross check.

Reply:
I'd have to double check but I think they should be consistent then? HPPC uses a random seed mix to avoid this problem:
https://issues.carrot2.org/browse/HPPC-103